access-sandbox-data.md

```diff
@@ -1,15 +1,13 @@
 ---
 layout: api-docs
-title:  "How to Access Test Claims Data"
-permalink: /access-test-claims-data
+title:  "How to Access Sandbox Data"
+permalink: /access-sandbox-data
 in-page-nav: true
 ---
 
 # {{ page.title }}
 
-The sandbox environment (sandbox.ab2d.cms.gov) is available to anyone who wants to test the API. First, you need to [get a bearer token]({{ '/get-a-bearer-token' | relative_url }}) with AB2D’s test credentials to access the sandbox. Then you can get test claims data by using a command line tool like curl or or writing your own client. 
-
-Accessing test data in the sandbox is a required step to receive production credentials. You’ll need to provide the AB2D team with the job ID from a successful run in the sandbox. [Learn more about onboarding]({{ '/onboarding' | relative_url }}).
+The sandbox environment (sandbox.ab2d.cms.gov) is available to anyone who wants to try the API. After retrieving sandbox data, follow the remaining steps to obtain [production access]({{ '/production-access' | relative_url }}).
```

Comment:
I agree I don't think it's necessary since we give pretty clear instructions in 1, 2, 3 part and onboarding/production access page. I'll proceed as recommended, but do we want anything else in the intro then? Otherwise it's just one sentence. 

I can either move the bearer token section back up into the intro or put back the line about how you can access sandbox data with a command line tool like curl. The latter is not strictly necessary since in the API Doc intro we let them know they need a client software tool to use the API, but could be a good reminder. Thoughts?

> "You need to get a bearer token in order to access the sandbox environment. Bearer tokens call the API and authorize use of the AB2D endpoints. [Learn how to get a bearer token]({{ '/get-a-bearer-token' | relative_url }})"

Reply:
I like mentioning the bearer token up front since that page isn't linked out in the steps below (wait, should we also link it out in the steps below when users input the token?)